new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -86,131 +77,135 @@ public void testInstallingOperator(@Namespaces(3) List<String> namespaces) {
     domainNamespace2 = namespaces.get(2);
 
     // Create a service account for the unique opNamespace
-    final String serviceAccountName = opNamespace + "-sa";
-    serviceAccount = new V1ServiceAccount()
+    logger.info("Creating service account");
+    String serviceAccountName = opNamespace + "-sa";
+    assertDoesNotThrow(() -> createServiceAccount(new V1ServiceAccount()
         .metadata(
             new V1ObjectMeta()
                 .namespace(opNamespace)
-                .name(serviceAccountName));
-
-    assertThatCode(
-        () -> createServiceAccount(serviceAccount))
-        .as("Test that createServiceAccount doesn not throw an exception")
-        .withFailMessage("createServiceAccount() threw an exception")
-        .doesNotThrowAnyException();
-    logger.info("Created service account: " + serviceAccountName);
+                .name(serviceAccountName))));
+    logger.info("Created service account: {0}", serviceAccountName);
+
+    String image = getOperatorImageName();
+    assertFalse(image.isEmpty(), "Operator image name can not be empty");
+    logger.info("Operator image name {0}", image);
+
+    // Create docker registry secret in the operator namespace to pull the image from repository
+    String repoRegistry = "dummy";
```

Comment:
is this all duplicated? maybe refactor it?

Reply:
I'm okay with doing the refactoring in a separate step because I was going to come after and have a third "strategy" for kind.

Reply:
I moved to TestConstants.java